src/content/terraform/install-landing.json

```diff
@@ -1,5 +1,12 @@
 {
-	"featuredTutorialsSlugs": [],
+	"featuredTutorialsSlugs": [
+            "terraform/tutorials/aws-get-started",
+	    "terraform/tutorials/azure-get-started",
+	    "terraform/tutorials/docker-get-started",
+	    "terraform/tutorials/gcp-get-started",
+	    "terraform/tutorials/oci-get-started",
+	    "terraform/tutorials/cloud-get-started"
```

Comment:
@laurapacilio Are these tutorials or collections? Only tutorials are supported here at the moment

Reply:
fixed!! thank you!